app/services/request-config.js

```diff
@@ -22,18 +20,15 @@ export default Service.extend({
       const { searchDebounceRate } = config.intervals;
       yield timeout(searchDebounceRate);
     }
-    try {
-      return yield this.store.queryRecord('request-config', { id: id, data });
-    } catch (e) {
+    return yield this.store.queryRecord('request-config', { id: id, data }).catch((e) => {
+      // TODO for some reason this still logs the 400 request as an error to the console
```

Comment:
no, i haven't been able to figure this out. i do think it would be good to silence these from the log because those error messages might be confusing.

Reply:
i have added that question to a new section "to be discussed/resolved" on https://github.com/travis-ci/travis-web/pull/2399 in order to keep track of this

Reply:
Hi @svenfuchs I might try to look into this, do you have a good URL / process to reproduce it?